<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sampling</class>
 <widget class="QDialog" name="Sampling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>168</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QComboBox" name="cbox_type">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>DownSampling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UniformSampling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RandomSampling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ReSampling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SamplingSurfaceNormal </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NormalSpaceSampling</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/res/icon/close-copy.svg</normaloff>:/res/icon/close-copy.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="DownSampling">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Radius</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dspin_r1">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="UniformSampling">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Radius: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dspin_r2">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RandomSampling">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Sample: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_sample1">
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Seed: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_seed1">
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ReSampling">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Radius: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dspin_r3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Order: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_order">
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SamplingSurfaceNormal">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Sample: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_sample2">
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Seed: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_seed2">
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Ratio: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dspin_ratio">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="NormalSpaceSampling">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Sample: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_sample3">
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Seed: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_seed3">
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Bin: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_bin">
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QCheckBox" name="check_reverse">
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/res/icon/reverse.svg</normaloff>:/res/icon/reverse.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_preview">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/res/icon/preview.svg</normaloff>:/res/icon/preview.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/res/icon/add.svg</normaloff>:/res/icon/add.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_apply">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/res/icon/apply1.svg</normaloff>:/res/icon/apply1.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reset">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/res/icon/reset.svg</normaloff>:/res/icon/reset.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbox_type</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
